src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -224,27 +84,18 @@ public static Font FromHfont(IntPtr hfont)
                 newStyle |= FontStyle.Bold;
             }
 
-            if (lf.lfHeight < 0)
-            {
-                newSize = lf.lfHeight * -1;
-            }
-            else
-            {
-                newSize = lf.lfHeight;
-            }
-
-            return (new Font(newObject, lf.lfFaceName, newStyle, newSize));
+            return new Font(newObject, lf.lfFaceName, newStyle, Math.Abs(lf.lfHeight));
         }
 
         public IntPtr ToHfont()
         {
-            if (fontObject == IntPtr.Zero)
+            if (_nativeFont == IntPtr.Zero)
                 throw new ArgumentException("Object has been disposed.");
```

Comment:
Now that you're in this cleanup should we change `"Object has been disposed."` to use a resource string?